src/routes/homepage-redesign-preview/index.svelte

```diff
@@ -1,106 +1,280 @@
+    margin-top: 2.5rem;
+    text-transform: uppercase;
+  }
+
+  // The following code creates a grid that has 1/3 and 2/3 column template
+  .one-two-column-grid {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 33% 67%;
+    grid-gap: 2rem;
+    padding: 0;
+    margin: 0;
+  }
+
+  .ae-logo-box {
+    padding: 14px;
+
+    img {
+      display: block;
+      margin: auto;
+    }
+  }
+
+  .ae-logo-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    align-items: flex-start;
+
+    @include media("<=medium") {
+      padding-top: 24px;
+    }
+  }
+
+  .cta-card {
+    @include media(">large") {
+      float: right;
+    }
+
+    @include media("<=medium") {
+      margin-top: 24px;
+    }
+  }
+</style>
+
 <svelte:head>
   <title>Cal-Adapt</title>
 </svelte:head>
 
+<!-- Banner -->
 <Banner
-  titleText="Cal-Adapt provides public access to climate data that contribute to
-    California’s Climate Change Assessments."
-  subtitleText="Cal-Adapt provides interactive, easy-to-use visualization and
-    data download tools for a broad range of stakeholders; a “pro” version for
-    users with highly technical, data-intensive needs; and links to state-funded
-    grants that support development of data for Cal-Adapt."
+  titleText="Explore and analyze climate data from California’s Climate Change Assessments"
+  subtitleText="Cal-Adapt provides the public, researchers, government agencies and industry stakeholders with essential data & tools for climate adaptation planning, building resiliency, and fostering community engagement."
+  bannerImg="/img/banners/yosemite_1600x540.jpg"
+  bannerImgMobile="/img/banners/yosemite_700x800.jpg"
+  overlayOpacity="{0.4}"
   titleFontSize="2rem"
   titleFontWeight="600"
+  iconPaths="{icons}"
   useOffset="{false}"
-  bannerImg="linear-gradient(180deg, #003850 0%, #067C74 100%)"
-  overlayOpacity="0"
-/>
+>
+  <!--   <div class="btn-container" slot="button">
+    <Button icon="{ArrowRight16}" href="/about" sapper:prefetch
+      >More about Cal-Adapt</Button
+    >
+  </div> -->
+  <div class="cta-card" slot="cta-card">
+    <Card
+      {...{
+        titleText: "Cal-Adapt is evolving!",
+        linkPath: `/help/`,
+        description:
+          "Learn about the Cal-Adapt enterprise and our mission to support California's climate change initiatives and preview our future plans.",
+        height: 18,
+        ctaText: "Read More",
+        bgColor: "rgba(255,255,255,0.5)",
+        border: "none",
+      }}
+    >
+      <div class="icon-circle bg-teal-60" slot="icon_slot">
+        <svelte:component this="{InformationFilled32}" />
+      </div>
+    </Card>
+  </div>
+</Banner>
 
 <div class="spacing--v-32"></div>
 
-<Grid>
-  <Row>
-    <Column lg="{{ span: 12 }}">
+<div class="bx--grid">
+  <div class="bx--row">
+    <div class="bx--col-lg-12 bx--col-md-8 bx--col-sm-4">
+      <!-- Fifth Assessment Notification -->
```

Comment:
@melijimenez11 Since we are planning to add the same notification to multiple pages, it might be good to move it out to a partial.